src/components/EmptyState/README.md

```diff
@@ -237,6 +237,29 @@ Stacked image over centered content and actions
 </EmptyState>
 ```
 
+### Empty state within a content context with centered layout and full width
```

Comment:
When in a content context, the layout is always centered. So we don't need to mention or add the `centeredLayout` prop in this example.

```suggestion
### Empty state with full width layout in a content context 
```